index.js

```diff
@@ -8,16 +8,19 @@ app.use(express.urlencoded({extended: true}));
 app.disable('etag');
 // app.use(morgan('combined'));
 
-/* app.use((req,res,next) => {
-    console.log("req.params =",req.params);
-    console.log("req.headers =",req.headers);
-    console.log("req.body =",req.body);
-    next();
-}); */
+app.use((req, res, next) => {
+  console.log('req.query =', req.query);
```

Comment:
does `morganBody` have anything related to this? Ig it will be better if we can use that only for this also

Reply:
Yes, morgan body do logs request query and request body, but it logs them at the end. So if we are modifying request query or body (we are modifying them), then `morganBody` will log these modified request not the original request received. 
So, logging request in the beginning in it's original state is often helpful.

Reply:
Ah, okay